stubs/docutils/docutils/transforms/__init__.pyi

```diff
@@ -1,14 +1,31 @@
-from _typeshed import Incomplete
+from collections.abc import Iterable
+from types import ModuleType
+from typing import Any
 
-from docutils.nodes import Node, document
+from docutils import ApplicationError, Component, TransformSpec, nodes
+from docutils.nodes import Node
+
+class TransformError(ApplicationError): ...
 
 class Transform:
-    def __init__(self, document: document, startnode: Node | None = None): ...
-    def __getattr__(self, name: str, /) -> Incomplete: ...
+    default_priority: int | None
+    document: nodes.document
+    startnode: Node | None
+    language: ModuleType
+    def __init__(self, document: nodes.document, startnode: Node | None = None) -> None: ...
+    def apply(self, **kwargs: Any) -> None: ...
```

Comment:
```suggestion
	@abc.abstractmethod
    def apply(self, **kwargs: Any) -> None: ...
```

Reply:
Since we can safely use abstractmethod without inheriting from ABC, we should mark the method as abstract (it raises a NotImplementedError at runtime).